src/loader/bgw_interface.c

```diff
@@ -2,21 +2,36 @@
 
 #include <miscadmin.h>
 #include <fmgr.h>
+
+#include "bgw_interface.h"
 #include "../compat.h"
 #include "bgw_counter.h"
 #include "bgw_message_queue.h"
-#include "bgw_interface.h"
+#include "../extension_constants.h"
 
 
 /* This is where versioned-extension facing functions live. They shouldn't live anywhere else. */
 
+int32 const ts_bgw_loader_api_version = 1;
```

Comment:
`int32 const` and `const int32` are equivalent, although, FWIW, I tend to prefer the latter, which is also what you did in the predeclaration